README.md

```diff
@@ -21,5 +21,11 @@ A summary of Reflex functions is available in the [quick reference](Quickref.md)
 
 ### Hacking
 
-Use the `./scripts/hack-on reflex` script in [Reflex Platform](https://github.com/reflex-frp/reflex-platform) to checkout the source code of `reflex` locally in `reflex-platform/reflex` directory.
-Then do modifications to the source in place, and use the `./try-reflex` or `./scripts/work-on` scripts to create the shell to test your changes.
+From the root of a [Reflex
```

Comment:
Maybe just recommend the one-liner from #397 if we're not going with the wrapper script

Reply:
I wasn't aware of #397 . Happy to withdraw this if we go for that instead.

Reply:
In the future, I like 1 sentence per line because 1) no one is going to read this in a terminal 2) it makes the diffs saner.